.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: release
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: set-up-jdk
+        uses: actions/setup-java@v1
+        with:
+          java-version: "11"
+          server-id: ossrh
+          settings-path: ${{ github.workspace }}
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: configure-git-user
+        run: |
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
```

Comment:
Perhaps now is the time to create a bot account for our organization(s). You should have all the permissions you need to create one. Maybe ask in #general on slack for people to suggest names.

Reply:
The only thing I don't have is the ability to create `@moderne.io` emails, which would be necessary for creating a `github` bot account with a proper git config username. Case-in-point, I created a github bot account using my `aaron@moderne.io` email here (https://github.com/openrewrite-ci-bot), but it wouldn't have it's own proper email association to it.

The git config is really just for display; it may be easier to just keep the `github-actions[bot]` username for now. Alternatively, we could put down `openrewrite` as the username, and `jkschneider@gmail.com` or `jonathan@moderne.io` as the email, and since we're using jonathan's `gpg` key, it'll show up as `Jonathan` as the committer 🤷

Reply:
github-actions is fine for now. We can create a service account later